### PR TITLE
Add pregnancy icon

### PR DESCRIPTION
**Pull request by clemens-msupply**
On special request from @DhanyaHerath and @roxy-dao :)

image

### CODE REVIEW

File: client/packages/host/public/medical-icons.css [hunk 1]
```diff
@@ -6,7 +6,7 @@
...
 }
```
Comment (clemens-msupply) [inline client/packages/host/public/medical-icons.css:10]: This auto formatting issue kept coming up so I finally comitted it...

**Comment by mark-prins** [comment]
@anisha-msupply looping you in, if you want to comment on the UI change.
Personally, I don't like the icon selected, but I don't want to be all curmudgeonly about such a subjective thing. The icon style is in keeping with the feather icon set and it is more 'preganancy' than the condom icon, so I say that we go for it.

**Review by mark-prins — APPROVED** [review]
👍 works fine thanks!
I wonder if the condom be removed from the css now. that would save us 4KB

**Comment by clemens-msupply** [comment]
Yes icon doesn't fit in 100%... Removed the condom, can re add if needed. Can the icons be loaded lazyly?

**Pull request merged**